conda.recipe/meta.yaml

```diff
@@ -18,15 +18,16 @@ requirements:
     build:
         - python
         - setuptools
-        - numpy x.x
         - cython
-        - hdf5
+        - numpy x.x
+        - msinttypes  # [win and py<35]
```

Comment:
@jswhit you will notice that Python 3.5 builds are fine because the Visual Studio used for Python 3.5 does have `stdint.h`

Here I am adding the `msinttypes` packages that should provide those. Not sure if I need to set some env var to ensure that `netcdf4-python` finds it.